pkg/capabilities/consensus/ocr3/datafeeds/feeds_aggregator.go

```diff
@@ -252,7 +247,7 @@ func (a *dataFeedsAggregator) extractSignersAndPayloads(observations map[ocrcomm
 			signers[signer]++
 		}
 		mins[meta.MinRequiredSignatures]++
-		payloads[nodeID] = triggerEvent.Payload
+		payloads[nodeID] = nodeObservations[0]
```

Comment:
nit: again the name "payload" is a little misleading to me now (see my other comment). After your change we're not saving payloads any more, we are saving whole events.